DEVELOPMENT.md

```diff
@@ -38,15 +38,14 @@ hatch run all:test
 
 ## Submitter environment
 
-Cinema4D does not support PYTHONPATH. We set DEADLINE_CLOUD_PYTHONPATH which the
-submitter and adaptor uses to set sys.path explictly and load deadline modules.
+Install deadline in a [supported search path](https://developers.maxon.net/docs/py/2024_0_0a/manuals/manual_py_libraries.html#python-interpreter-bound-search-paths) or set a [custom interpreter path](https://developers.maxon.net/docs/py/2024_0_0a/manuals/manual_py_libraries.html#custom-interpreter-bound-search-paths) to the deadline install location.
 
 - install deadline-cloud with pyside
 - set env below
 
 ```
 # deadline-cloud lib with pyside
-export DEADLINE_CLOUD_PYTHONPATH="/path/to/deadline-cloud/site-packages"
+export C4DPYTHONPATH311 "/path/to/deadline-cloud/site-packages"
```

Comment:
Missing `=`? Also this may have some merge conflicts with[ this PR](https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/pull/59/files).  Same issue below

Reply:
Also any reason this is suffixed with 311? I was able to get it working on a machine with Python 3.12 as well.

Reply:
Yes, thanks, missing = is a typo.
Reason for suffix is dev docs seem to indicate C4DPYTHONPATH is for python 2.7 only, I wouldn't be surprised if 3.12 worked too, especially pure python modules: https://developers.maxon.net/docs/py/2024_0_0a/manuals/manual_py_libraries.html#custom-interpreter-bound-search-paths:~:text=%3C%3D%20C4D_VERSION)-,C4DPYTHONPATH311,-Note%20To%20register

Reply:
Thanks for your clarification!